devicetypes/sky-nie/evalogik-door-window-sensor.src/evalogik-door-window-sensor.groovy

```diff
@@ -0,0 +1,585 @@
+/**
+ *     Evalogik Door/Window Sensor v1.0.5
+ *
+ *  	Models: MSE30Z
+ *
+ *  Author:
+ *   winnie (sky-nie)
+ *
+ *	Documentation:
+ *
+ *  Changelog:
+ *
+ *    1.0.5 (07/28/2021)
+ *     - omitted  all the parameters related to associations group
+ *
+ *    1.0.4 (07/16/2021)
+ *     - Syntax format compliance adjustment
+ *     - fixed a bug for order repeated
+ *
+ *    1.0.3 (07/16/2021)
+ *     - change lastBatteryReport to record the time of fresh battery
+ *     - add lastBattery to record the battery value
+ *
+ *    1.0.2 (07/15/2021)
+ *      - update ConfigParams as product designed
+ *      - update DTH name as product designed
+ *
+ *    1.0.1 (07/13/2021)
+ *      - Syntax format compliance adjustment
+ *      - delete dummy code
+ *
+ *    1.0.0 (04/26/2021)
+ *      - Initial Release
+ *
+ * Reference：
+ *   https://community.smartthings.com/t/release-aeotec-trisensor/140556?u=krlaframboise
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+final  int NOTIFICATION_TYPE_ACCESS_CONTROL = 0x06
+final  int NOTIFICATION_TYPE_HOME_SECURITY = 0x07
+
+final  int NOTIFICATION_EVENT_DOOR_WINDOW_OPEN = 0x16
+final  int NOTIFICATION_EVENT_DOOR_WINDOW_CLOSED = 0x17
+
+final  int NOTIFICATION_EVENT_STATE_IDLE = 0x00
+final  int NOTIFICATION_EVENT_INSTRUSION_WITH_LOCATION = 0x01
+final  int NOTIFICATION_EVENT_INSTRUSION = 0x02
+final  int NOTIFICATION_EVENT_TEMPERING = 0x03
+
+metadata {
+	definition(name: "Evalogik Door/Window Sensor", namespace: "sky-nie", author: "winnie", ocfDeviceType: "x.com.st.d.sensor.contact", executeCommandsLocally: false, genericHandler: "Z-Wave") {
```

Comment:
remove the `genericHandler` and `executeCommandsLocally` flags

Reply:
Ok, I  remove them.